quill-jdbc-zio/src/main/scala/io/getquill/context/ZioJdbc.scala

```diff
@@ -151,14 +151,16 @@ object ZioJdbc {
         .ignore
     )
 
-  private[getquill] val streamBlocker: ZStream[Any, Nothing, Any] =
-    ZStream.scoped {
-      for {
-        executor         <- ZIO.executor
-        blockingExecutor <- ZIO.blockingExecutor
-        _                <- ZIO.acquireRelease(ZIO.shift(blockingExecutor))(_ => ZIO.shift(executor))
-      } yield ()
-    }
```

Comment:
This was probably incorrect and was probably not doing anything.

The `ZIO.acquireRelease(ZIO.shift(blockingExecutor))(_ => ZIO.shift(executor))` takes action in the returned `Scope`. 
Here, we were opening and closing a new Scope for the execution of this code. Which means that, after this code, the executor would already have been shifted back.

To include any code execution on the Blocking executor, we need to return the `Scope` needs to the caller. So that the caller has the control over when the shift back happen.